hashicorp/consul
# Hide the hcp link banner behind an env variable

### Description
With the 1.18 RC release getting cut tomorrow, we want to hide the HCP link banner - as it won't be done until the 1.18.0 release.

### Testing & Reproduction steps
- Launch app normally - see no banner
- Launch against [consul-setup](https://github.com/WenInCode/consul-setup/tree/main/setups/peering) - see no banner
- Locally run `Scenario('CONSUL_HCP_LINK_ENABLED=true')` and see the banner exists now

### Links
- [Jira](https://hashicorp.atlassian.net/browse/CC-7387)

### References
![image](https://github.com/hashicorp/consul/assets/4359781/b3d50b91-50c5-4dd3-af7b-ab3fe9a6005a)
